app.js

```diff
@@ -3,163 +3,254 @@ import { app, errorHandler } from 'mu';
 import bodyParser from 'body-parser';
 
 import { waitForDatabase } from './lib/util/database';
-import { FILES, FormVersionService } from './lib/services/form-version-service';
-import { FormManagementService } from './lib/services/form-management-service';
-import { uriToPath } from './lib/util/file';
 import { Model } from './lib/model-mapper/entities/model';
 import { ModelMapper } from './lib/model-mapper/model-mapper';
+import { SemanticFormManagement } from './lib/services/semantic-form-management';
+import { ConfigurationFiles } from './lib/services/configuration-files';
+import { MetaFiles } from './lib/services/meta-files';
+import { SourceDataExtractor } from './lib/services/source-data-extractor';
+import { DEV_ENV, SEMANTIC_FORM_RESOURCE_BASE, SERVICE_NAME } from './env';
+import { MetaDataExtractor } from './lib/services/meta-data-extractor';
+
+/**
+ * Setup and API.
+ */
 
 app.use(bodyParser.json({
   type: function(req) {
     return /^application\/json/.test(req.get('content-type'));
   },
 }));
 
+/**
+ * Hello world (basic is alive test).
+ */
 app.get('/', function(req, res) {
-  const message = 'Hey there, you have reached subsidy-applications-management-service! Seems like I\'m doing just fine! :)';
+  const message = `Hey there, you have reached ${SERVICE_NAME}! Seems like I\'m doing just fine, have a nice day! :)`;
   res.send(message);
 });
 
-let formManagementService;
-let versionService;
-
-waitForDatabase().then(async () => {
-  versionService = await new FormVersionService().init();
-  formManagementService = new FormManagementService(versionService);
-});
+let config_files;
+let meta_files;
```

Comment:
Is there a reason why you don't use camelCase here ?

Reply:
no, me just not knowing what to name it and using a placeholder. I'll rename it!